examples/experimental_component/src/button.rs

```diff
@@ -0,0 +1,97 @@
+#![allow(dead_code)]
+
+use seed::{prelude::*, *};
+use std::borrow::Cow;
+use std::rc::Rc;
+
+pub struct Button<S> {
+    pub label: S,
+}
+
+impl<S: Into<Cow<'static, str>>> Button<S> {
+    pub fn into_component<Ms>(self) -> ButtonComponent<Ms> {
+        ButtonComponent {
+            label: self.label.into(),
+            outlined: false,
+            disabled: false,
+            on_clicks: Vec::new(),
+        }
+    }
+}
+
+#[allow(clippy::module_name_repetitions)]
+pub struct ButtonComponent<Ms: 'static> {
```

Comment:
This is a bit of an unsightly wart for those who use clippy. Not entirely sure what to do about it, but probably either the `trait` or the naming convention here needs to be renamed.